app/pathogen/arbovirus/dashboard/(visualizations)/country-seroprevalence-comparison-scatter-plot.tsx

```diff
@@ -0,0 +1,204 @@
+            (dataPoint.seroprevalenceCalculated95CIUpper * 100).toFixed(1)
+          ),
+          seroprevalenceError: [
+            parseFloat(
+              (
+                dataPoint.seroprevalence * 100 -
+                dataPoint.seroprevalenceCalculated95CILower * 100
+              ).toFixed(1)
+            ),
+            parseFloat(
+              (
+                dataPoint.seroprevalenceCalculated95CIUpper * 100 -
+                dataPoint.seroprevalence * 100
+              ).toFixed(1)
+            ),
+          ],
+        })),
+    [state.filteredData]
+  );
+
+  const { rechartsData } = useMemo(
+    () =>
+      groupDataForRecharts({
+        data: stateFilteredDataWithEstimateNumbers,
+        primaryGroupingFunction: (dataPoint) =>
+          dataPoint.pathogen as arbovirusesSF,
+        primaryGroupingSortFunction: (pathogenA, pathogenB) =>
+          pathogenA !== pathogenB ? (pathogenA < pathogenB ? -1 : 1) : 0,
+        secondaryGroupingFunction: (dataPoint) => dataPoint.country as string,
+        secondaryGroupingSortFunction: (countryA, countryB) =>
+          countryA !== countryA ? (countryA < countryB ? -1 : 1) : 0,
+        transformOutputValue: (data) => data,
+      }),
+    [stateFilteredDataWithEstimateNumbers]
+  );
+
+  const pathogensAvailable = useMemo(
+    () => uniq(rechartsData.map((dataPoint) => dataPoint.primaryKey)),
+    [rechartsData]
+  );
+  const countriesAvailable = useMemo(
+    () =>
+      uniq(
+        rechartsData.flatMap((dataPoint) =>
+          typedObjectKeys(dataPoint).filter((value) => value !== "primaryKey")
+        )
+      ),
+    [rechartsData]
+  );
+
+  if (pathogensAvailable.length > 1) {
+    return (
+      <div>
+        <p>
+          This graph cannot be shown when the data contains more than one
+          arbovirus. If you&apos;re interested in generating this graph, please use
+          the filter to the left to filter out data.
+        </p>
+      </div>
+    );
+  }
+
+  if (countriesAvailable.length > 2) {
+    return (
+      <div>
+        <p>
+          This graph cannot be shown when the data contains more than two
+          countries. If you&apos;re interested in generating this graph, please use
+          the filter to the left to filter out data.
+        </p>
+      </div>
+    );
+  }
+
+  const dataGroupedByCountry =
+    rechartsData[0] ?? countriesAvailable.map((country) => ({ [country]: [] }));
+
+  return (
+    <div className="w-full h-full flex">
+      {countriesAvailable.map((country, index) => (
+        <ResponsiveContainer
+          width={countriesAvailable.length === 1 ? "100%" : "50%"}
+          key={`country-seroprevalence-comparison-scatter-plot-${country}`}
+          height={"100%"}
+          className={index > 0 ? "pl-4" : ""}
+        >
+          <ScatterChart
+            width={730}
+            height={250}
+            margin={{ bottom: 16, left: 8, top: 50, right: 10 }}
+          >
+            <text
```

Comment:
Keep in mind recharts renders an svg so I can't just throw `<h2>`s and `<p>`s in here. That's why I had to do what I had to do.